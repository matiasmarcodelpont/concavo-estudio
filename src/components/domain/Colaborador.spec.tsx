import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

import { Colaborador } from './Colaborador'

const colaborador = {
  slug: 'elegance',
  name: 'Elegance',
  contact: 'contact@contact.com',
}

describe('Colaborador', () => {
  it('matches snapshot', () => {
    const { container } = render(<Colaborador {...colaborador} />)
    expect(container).toMatchSnapshot()
  })

  it('should render the colaborador image with the alt text', () => {
    const { getByAltText } = render(<Colaborador {...colaborador} />)

    const image = getByAltText(colaborador.name)
    expect(image).toBeInTheDocument()
  })
})
