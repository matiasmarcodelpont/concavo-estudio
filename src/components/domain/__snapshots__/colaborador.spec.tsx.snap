// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainColaborador matches snapshot 1`] = `
<div>
  <div
    class="flex flex-col gap-3"
  >
    <div
      class="w-[300px] h-[150px] relative"
    >
      <img
        alt="Elegancee"
        class="max-w-[300px] max-h-[150px] grayscale"
        data-nimg="fill"
        decoding="async"
        loading="lazy"
        sizes="(max-width: 768px) 100vw, 33vw"
        src="/_next/image?url=%2Fcolaboradores%2Felegance.png&w=3840&q=75"
        srcset="/_next/image?url=%2Fcolaboradores%2Felegance.png&w=256&q=75 256w, /_next/image?url=%2Fcolaboradores%2Felegance.png&w=384&q=75 384w, /_next/image?url=%2Fcolaboradores%2Felegance.png&w=640&q=75 640w, /_next/image?url=%2Fcolaboradores%2Felegance.png&w=750&q=75 750w, /_next/image?url=%2Fcolaboradores%2Felegance.png&w=828&q=75 828w, /_next/image?url=%2Fcolaboradores%2Felegance.png&w=1080&q=75 1080w, /_next/image?url=%2Fcolaboradores%2Felegance.png&w=1200&q=75 1200w, /_next/image?url=%2Fcolaboradores%2Felegance.png&w=1920&q=75 1920w, /_next/image?url=%2Fcolaboradores%2Felegance.png&w=2048&q=75 2048w, /_next/image?url=%2Fcolaboradores%2Felegance.png&w=3840&q=75 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: contain; color: transparent;"
      />
    </div>
    <p
      class="text-xs text-center text-darkGray"
    >
      Líder en revestimientos de suelos elegantes y duraderos para hogares y negocios. Calidad, innovación y compromiso con la sostenibilidad nos distinguen.
    </p>
    <div
      class="flex flex-col gap-1"
    >
      <p
        class="text-xs text-center font-semibold text-darkGray"
      >
        Avenida del Diseño 123, Ciudad Elegante, España
      </p>
      <div
        class="text-center flex justify-center gap-2"
      >
        <a
          class="text-xs text-center font-semibold text-blue hover:underline"
          href="mailto:info@pisoselegance.com"
          target="_blank"
        >
          info@pisoselegance.com
        </a>
        <div
          aria-orientation="vertical"
          class="w-[1px] bg-darkGray"
          data-orientation="vertical"
          role="separator"
        />
        <a
          class="text-xs text-center font-semibold text-blue hover:underline"
          href="https://www.pisoselegance.es"
          target="_blank"
        >
          www.pisoselegance.es
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`StandardColaborador matches snapshot 1`] = `
<div>
  <div
    class="w-[300px] h-[150px] relative"
  >
    <img
      alt="Muebles de Arte"
      class="max-w-[300px] max-h-[150px] grayscale"
      data-nimg="fill"
      decoding="async"
      loading="lazy"
      sizes="(max-width: 768px) 100vw, 33vw"
      src="/_next/image?url=%2Fcolaboradores%2Fmuebles-arte.png&w=3840&q=75"
      srcset="/_next/image?url=%2Fcolaboradores%2Fmuebles-arte.png&w=256&q=75 256w, /_next/image?url=%2Fcolaboradores%2Fmuebles-arte.png&w=384&q=75 384w, /_next/image?url=%2Fcolaboradores%2Fmuebles-arte.png&w=640&q=75 640w, /_next/image?url=%2Fcolaboradores%2Fmuebles-arte.png&w=750&q=75 750w, /_next/image?url=%2Fcolaboradores%2Fmuebles-arte.png&w=828&q=75 828w, /_next/image?url=%2Fcolaboradores%2Fmuebles-arte.png&w=1080&q=75 1080w, /_next/image?url=%2Fcolaboradores%2Fmuebles-arte.png&w=1200&q=75 1200w, /_next/image?url=%2Fcolaboradores%2Fmuebles-arte.png&w=1920&q=75 1920w, /_next/image?url=%2Fcolaboradores%2Fmuebles-arte.png&w=2048&q=75 2048w, /_next/image?url=%2Fcolaboradores%2Fmuebles-arte.png&w=3840&q=75 3840w"
      style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: contain; color: transparent;"
    />
  </div>
</div>
`;
